src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
 
 ### New Features
 - Added partial support for `SeriesGroupBy.apply` (where the `SeriesGrouBy` is obtained through `DataFrameGroupBy.__getitem__`).
+- Added `DataFrame.cache` and `Series.cache` methods for users to cache computationally intensive intermediate results for reuse.
```

Comment:
I think we need a `performance.rst` file, in `docs/source/modin/performance.rst` and I think this should be the first entry. We might want to add more later.